### PR TITLE
Switch to 0.7 on master

### PR DESCRIPTION
**Pull request by daschw**

### CODE REVIEW

**Comment by daschw** [comment]
The test will fail until https://github.com/JuliaImages/ImageMetadata.jl/issues/25 is fixed.

File: REQUIRE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 0.7
```
Comment (fredrikekre) [inline REQUIRE]: Set to `julia 0.7-alpha`.

File: REQUIRE [hunk 2]
```diff
@@ -14,3 +14,4 @@ NaNMath
 Requires
 Contour
 GR 0.31.0
+Dates
```
Comment (fredrikekre) [inline REQUIRE]: `Dates` should not be in `REQUIRE`.

**Comment by daschw** [comment]
OK, thanks a lot for the hints @fredrikekre !

**Comment by fredrikekre** [comment]
I have some additional fixes that I will push soonish

**Comment by fredrikekre** [comment]
Ok, you can push your changes and I can rebase on top of that if I have something to add.

**Pull request merged**